tools/pythonpkg/duckdb/experimental/spark/context.py

```diff
@@ -9,6 +9,11 @@
 class SparkContext:
     def __init__(self, master: str):
         self._connection = duckdb.connect(':memory:')
+        # We need this to align the ordering of ascending with Spark. For descending,
+        # Spark uses nulls last. Hence, in the API, we need to take care of that.
+        # Setting the ordering at least correctly for ascending is relevant
+        # for DataFrame.sort, see the comments there for more information.
+        self._connection.execute("set default_null_order='nulls first'")
```

Comment:
Perhaps set this to `nulls_first_on_asc_last_on_desc` instead to fully match Sparks' behavior?

Reply:
Oh, that's exactly what I was looking for, thanks! I'll ping you once I've implemented it. That will simplify some things.

I'll see if I manage to open a PR to update the docs as well:
<img width="871" alt="image" src="https://github.com/user-attachments/assets/fa69764b-bec4-4957-81ee-123e95d32bba">

Reply:
Done (was able to revert most of the changes to `column.py` and `dataframe.py` thanks to this). 

Also opened a PR to update the docs https://github.com/duckdb/duckdb-web/pull/4034

PR is ready from my side.